LoRaEngine/test/LoRaWan.IntegrationTest/C2DMessageTest.cs

```diff
@@ -108,6 +109,21 @@ public async Task Test_OTAA_Confirmed_Receives_C2D_Message()
                 // +CMSG: ACK Received
                 await AssertUtils.ContainsWithRetriesAsync("+CMSG: ACK Received", this.ArduinoDevice.SerialLogs);
 
+                // Check that RXDelay was correctly used
+                if (this.ArduinoDevice.SerialLogs.Where(x => x.StartsWith("+CMSG: RXWIN1")).Count() > 0)
+                {
+                    await this.TestFixtureCi.CheckAnswerTimingAsync(2000000, false);
+                }
+                else
+                if (this.ArduinoDevice.SerialLogs.Where(x => x.StartsWith("+CMSG: RXWIN2")).Count() > 0)
+                {
+                    await this.TestFixtureCi.CheckAnswerTimingAsync(2000000, false);
```

Comment:
shouldn't the 2nd parameter be true here?

Reply:
Yes this is an error, thanks for spotting!